mne/tests/test_import_nesting.py

```diff
@@ -2,18 +2,14 @@
 #
 # License: BSD-3-Clause
 
+import os
 import sys
+import pytest
 from mne.utils import run_subprocess
 
-# To keep `import mne` time down, we nest most imports, including those of
-# required dependencies (e.g., pooch, tqdm, matplotlib). SciPy submodules
-# (esp. linalg) can take a while to load, so these imports must be nested, too.
-# NumPy is the exception -- it can be imported directly at the top of files.
-# N.B. that jinja2 is imported directly in mne/html_templates, so all imports
-# of mne.html_templates must be nested to achieve the jinja2 import being
-# nested during import of mne.
-#
-# This test ensures that we don't accidentally un-nest any of these imports.
+eager_import = os.getenv("EAGER_IMPORT", "")
+
+# This test ensures that modules are lazily loaded.
```

Comment:
would be good to update the test name and parts of `run_script` to more clearly reflect the new situation --- but that can be a sunday hack later for someone else if you want.